src/applications/static-pages/addIdToSubHeadings.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Add unique ID to H2s and H3s that aren't in WYSIWYG or accordion buttons
+ */
+
+export function uniqueId() {
+  return `id-${Math.random()
+    .toString(36)
+    .substr(2, 16)}`;
+}
+
+export function generateHeadingIds(headings) {
```

Comment:
We could do this in a metalsmith plugin and avoid the client side JS, I think. This is an example: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/plugins/update-external-links.js

In that plugin we loop through all the html files, load them into cheerio, query the html for the elements we're looking for, then update them.

Reply:
Made this part of the build and removed client side js